packages/discovery/src/discovery/runDiscovery.ts

```diff
@@ -56,6 +52,12 @@ export async function runDiscovery(
     blockNumber,
     config.chain.rpcGetLogsMaxRange,
   )
+
+  const projectMeta = await configReader.readMeta(
+    config.project,
+    config.chain.name,
+    true,
+  )
```

Comment:
Why do we skip the inlining of templates here? There is nothing to indicate the reason for it. The `boolean` as argument nature of this code feels hacky and is hard to understand. Because now I don't know if it should be also set to `true` in other places that call `readMeta()`. If we really need to do this, I'd argue that it's better to create two functions - `readMetaWithoutInlining()` and just `readMeta()`